package.json

```diff
@@ -2,7 +2,7 @@
   "name": "angular2-aot-webpack",
   "version": "1.0.0",
   "description": "Angular 2 AOT (Ahead Of Time) offline compilation example with Webpack",
-  "main": "index.js",
+  "main": "build/modules/main.module.js",
```

Comment:
This is so that you can import the main lib module directly using:

`import { MainModule } from 'my-library';`

Instead of having to specify a path to the `main.module.js` file.